modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/TpchBenchmark.java

```diff
@@ -54,7 +51,7 @@
 @BenchmarkMode(Mode.AverageTime)
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
 @SuppressWarnings({"WeakerAccess", "unused"})
-public class TpchBenchmark extends AbstractMultiNodeBenchmark {
+public class TpchBenchmark extends AbstractTpcBenchmark {
```

Comment:
please fix javadoc below, due to fsync mode always is off from now

Reply:
done